app/controllers/v0/burial_claims_controller.rb

```diff
@@ -4,26 +4,31 @@
 require 'burials/monitor'
 
```

Comment:
Is the BurialClaimsController missing`require 'common/exceptions/validation_errors'`? ClaimsBaseController had `require 'common/exceptions/validation_errors'` and ApplicationController does not